python/hail/docs/_templates/layout.html

```diff
@@ -129,13 +129,7 @@
             </ul>
           </li>
             <li id="forum" class="nav-item"><a href="http://discuss.hail.is">FORUM</a></li>
-            <li id="chat" class="nav-item dropdown">
-            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">CHAT<span class="caret"></span></a>
-            <ul class="dropdown-menu dropdown-menu-left">
-              <li><a class="dropdown-item" href="https://gitter.im/hail-is/hail">General</a></li>
-              <li><a class="dropdown-item" href="https://gitter.im/hail-is/hail-dev">Developers</a></li>
-            </ul>
-          </li>
+            <li id="chat" class="nav-item"><a href="http://hail.zulipchat.com">CHAT</a></li>
```

Comment:
Unfortunately, I couldn't test the link on the docs preview. Got an error message relating to security. Can you test it works locally?

Reply:
it works!